### ISSUE TITLE
"Go Go Plant 2" doesn't do anything when the play button is clicked

### ISSUE THREAD

**Issue by grishka** [ruffle-rs/ruffle#2320]
Log after it initializes and gets to the screen with the button: (there are many errors about "unknown tag codes" 253 and 255, and also "Stage.showMenu: unimplemented")
[ruffle.rs-1610035505202.log](https://github.com/ruffle-rs/ruffle/files/5782686/ruffle.rs-1610035505202.log)
Then there are these errors when you click the button:
```
Uncaught Error: closure invoked recursively or destroyed already
    at Object.Wr (ruffle_web_bg.js:1686)
    at __wbindgen_throw (wasm chunk loading:1364)
    at wasm_bindgen::throw_str::hfa971da691102d09 (/demo/827bed83bde6126886bf.module.wasm:wasm-function[4958]:0x34ec53)
    at <dyn core::ops::function::FnMut<(A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::h21ecf576ef0c1ac8 (/demo/827bed83bde6126886bf.module.wasm:wasm-function[4642]:0x34d328)
    at P (ruffle_web_bg.js:240)
    at f (ruffle_web_bg.js:205)

Uncaught Error: closure invoked recursively or destroyed already
    at Object.Wr (ruffle_web_bg.js:1686)
    at __wbindgen_throw (wasm chunk loading:1364)
    at wasm_bindgen::throw_str::hfa971da691102d09 (/demo/827bed83bde6126886bf.module.wasm:wasm-function[4958]:0x34ec53)
    at <dyn core::ops::function::FnMut<(A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::h21ecf576ef0c1ac8 (/demo/827bed83bde6126886bf.module.wasm:wasm-function[4642]:0x34d328)
    at P (ruffle_web_bg.js:240)
    at HTMLElement.f (ruffle_web_bg.js:205)
```
SWF:
[go-go-plant-2.swf.zip](https://github.com/ruffle-rs/ruffle/files/5782672/go-go-plant-2.swf.zip)

OS: macOS 10.14.6, browser: Vivaldi 3.5.2115.81 (aka Chrome 87.0.4280.107)

**Comment by Toad06** [comment]
Thanks for the report. The source code is heavily obfuscated which is causing the "unknown tag codes" errors. Ruffle will need to handle that.

**Comment by grishka** [comment]
I just retested it now, and as of the 2021-01-24 nightly build, this game is fully playable. Save for the shadows (yeah I know bitmap effects), it works as good as in the Adobe flash player.

**Comment by Herschel** [comment]
Fixed by #2636. Thanks for testing!
